src/vmm/src/devices/virtio/block/virtio/metrics.rs

```diff
@@ -178,11 +178,17 @@ pub struct BlockDeviceMetrics {
     pub read_count: SharedIncMetric,
     /// Number of successful write operations.
     pub write_count: SharedIncMetric,
+    /// Duration of all read operations.
```

Comment:
If we use [LatencyAggregateMetrics](https://github.com/sudanl0/firecracker/blob/be25214c564812babbc614ad2e4077afd0bba4a3/src/vmm/src/logger/metrics.rs#L731) instead of SharedIncMetrics we can capture min/max and sum of the read_duration_us. Do you think there is any advantage of exporting main/max in addition to the sum that the code currently exports?

Reply:
yes, LatencyAggregateMetrics looks like a good fit to that, thanks for proposing, made use of them. The only thing is it was not easy to use it with tap read due to borrowing rules, so I decided to skip that metric for now, especially I only add queue len for Tx, not for Rx.